prow/tide/tide.go

```diff
@@ -1011,9 +1031,13 @@ func (c *Controller) pickBatch(sp subpool, cc map[int]contextChecker) ([]PullReq
 	// we must choose the oldest PRs for the batch
 	sort.Slice(sp.prs, func(i, j int) bool { return sp.prs[i].Number < sp.prs[j].Number })
 
+	batchSkipTests := false
+	if c.config().Tide.BatchSkipTests != nil && *c.config().Tide.BatchSkipTests {
+		batchSkipTests = true
```

Comment:
```suggestion
	var batchSkipTests bool
	if c.config().Tide.BatchSkipTests != nil {
		batchSkipTests = *c.config().Tide.BatchSkipTests 
```

is IMO a bit more idiomatic and less wordy, but I would not even be afraid of doing this in a single line (`bst := (ptr != nil && *ptr`)

Reply:
will try the one liner thanks!